From 85ab882acd4861808d412f57ca98dc3c78150bf9 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Mon, 16 Apr 2012 21:42:42 -0300
Subject: [PATCH 35/40] 4.4.11 Cuenca de Sholazar: El salvador de los
 Caminamoho

Support for quest "The Mosswalker Savior" from Oracles/Frenzyheart quest chain
---
 .../2012_04_16_112_world_the_mosswalker_savior.sql |   42 ++++++++++++++++++++
 1 files changed, 42 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_16_112_world_the_mosswalker_savior.sql

diff --git a/sql/updates/world/reanemu/2012_04_16_112_world_the_mosswalker_savior.sql b/sql/updates/world/reanemu/2012_04_16_112_world_the_mosswalker_savior.sql
new file mode 100644
index 0000000..90e7e97
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_16_112_world_the_mosswalker_savior.sql
@@ -0,0 +1,42 @@
+-- The Mosswalker Savior
+SET @CHAIN := 54324; -- spell chain visual
+SET @CORPSE := 28641;
+SET @ORACLE := 28113;
+
+DELETE FROM `creature` WHERE `id`=@ORACLE;
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry` IN (@CORPSE,@ORACLE);
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (@CORPSE,@ORACLE) AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@CORPSE,0,0,0,11,0,100,0,0,571,3711,0,12,@ORACLE,8,0,0,0,0,1,0,0,0,0,0,0,0,'Blighted Corpse - Spawn - Summon Mosswalker Victim'),
+(@CORPSE,0,1,0,6,0,100,0,0,0,0,0,45,0,1,0,0,0,0,19,@ORACLE,20,0,0,0,0,0,'Blighted Corpse - Death - Set data Mosswalker Victim'),
+(@ORACLE,0,0,1,11,0,100,0,8,571,3711,0,86,@CHAIN,0,19,@CORPSE,20,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Spawn - Cross cast chain visual on self'),
+(@ORACLE,0,1,0,61,0,100,0,0,0,0,0,29,3,0,0,0,0,0,19,@CORPSE,20,0,0,0,0,0,'Mosswalker Victim - Spawn - Follow Closest Blighted Corpse'),
+(@ORACLE,0,2,3,38,0,100,1,0,1,0,0,45,0,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Data set - Set data self'),
+(@ORACLE,0,3,4,61,0,100,0,0,0,0,0,19,333554432,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Data set - Remove unitflags'),
+(@ORACLE,0,4,5,61,0,100,0,0,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Data set - Say text'),
+(@ORACLE,0,5,0,61,0,100,0,0,0,0,0,41,15000,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Data set - Despawn after 15 secs'),
+(@ORACLE,0,6,7,62,0,100,0,9728,0,0,0,1,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Gossip select - Say text'),
+(@ORACLE,0,7,8,61,0,100,0,0,0,0,0,72,0,0,0,0,0,0,7,0,0,0,0,0,0,0,'Mosswalker Victim - Gossip select - close gossip'),
+(@ORACLE,0,8,9,61,0,100,0,0,0,0,0,33,28644,0,0,0,0,0,7,0,0,0,0,0,0,0,'Mosswalker Victim - Gossip select - call killed monster'),
+(@ORACLE,0,9,10,61,0,100,0,0,0,0,0,83,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Gossip select - remove npc flags'),
+(@ORACLE,0,10,0,61,0,100,0,0,0,0,0,41,4000,0,0,0,0,0,1,0,0,0,0,0,0,0,'Mosswalker Victim - Gossip select - despawn');
+
+DELETE FROM `gossip_menu_option` WHERE `menu_id`=9728;
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`) VALUES
+(9728,0,0,"<Check for a pulse...>",1,1,0,0,0,0,'');
+
+DELETE FROM `creature_text` WHERE `entry`=@ORACLE;
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`comment`) VALUES
+(@ORACLE,0,0,"The mosswalker victim groans in pain",16,0,100,0,0,0,'Mosswalker victim'),
+(@ORACLE,1,0,"Help more mosswalkers, okay?",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,1,"I do something bad? I sorry....",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,2,"I give more shinies to shrines! I promise! I sorry!",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,3,"Maybe you make weather better too?",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,4,"Please take... my shinies. All done...",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,5,"Thank you! You good!",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,6,"Thank... you.",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,7,"Use my shinies... make weather good again... make undead things go away.",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,8,"We gave shinies to shrine... we not greedy... why this happen?",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,9,"We not do anything... to them... I no understand.",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,10,"We saved. You nice, dryskin.",12,0,100,1,0,0,'Mosswalker victim'),
+(@ORACLE,1,11,"You save us! Yay for you!",12,0,100,1,0,0,'Mosswalker victim');
\ No newline at end of file
-- 
1.7.8.msysgit.0

